src/en/config-LXD.md

```diff
@@ -0,0 +1,204 @@
+Title: LXD local provider
+
+
+# Overview
+
+The LXD local provider is the most efficient and featureful way to use Juju
+locally on Ubuntu. It leverages [LXD](https://linuxcontainers.org/lxd/) to
+enable multiple Juju-deployed services within a single operating system.
+Immediate uses include:
+
+ - Evaluation of Juju
+ - Evaluation of deployed software
+ - Experimentation with various service configurations
+ - Charm development
+
+It is a replacement for the traditional LXC local provider and has some
+notable differences:
+
+ - The state server is no longeer localhost but just another LXC container.
+   The benefits of keeping installed software/files isolated in a container far
+   outweigh the extra time needed to create this container.
+ - Due to the above, superuser privileges (sudo) are no longer required.
+ - There is no more "Juju cache" for images. LXD now has its own cache
+   (`/var/lib/lxd/images`) which is separate from the traditional LXC cache
+   (`/var/cache/lxc`).
+
+Although this document does provide enough information to get you running with
+LXD itself (this document is ultimately about running LXD with Juju) it does
+so assuming you are setting up LXD for the first time.
+
+**Note:** If you are running iptables (firewall) or even an iptables frontend
```

Comment:
To be clear, the disabling of the firewall was meant as a temporary troubleshooting technique only. That should be clarified.